ASWF/meetings/working_groups/configs/2020-11-03.md

```diff
@@ -0,0 +1,147 @@
+      when merged will remove a lot of complexity. We can influence change 
+      there similarly to how ACES influences OCIO change.
+    - Michael: Last meeting we discussed that the input BuiltinTransforms were 
+      largely present, but needed output transforms to proceed. Is there an 
+      expected timeline for that work Doug?
+    - Doug: Will try to get something before next meeting.
+    - Mark: OCIO v2 has display transforms. Are we making use of that? Some 
+      transforms have OCES space, but PQ goes straight from scene linear. This 
+      config is supposed to represent all the features of OCIO v2. should we 
+      make use of both?
+    - Doug: Yes, will include this in proposal, but think will make use of 
+      display color spaces, so output transforms will be view transform plus 
+      display transform. OCES wouldn't have an interface. More recent ACES 
+      transforms have moved away from it. OCES was interesting theoretical 
+      idea, providing a break point between points of transform that proved to 
+      be more confusing. Would not use OCES.
+    - Mark: What would become the display space? The generator draws 
+      connections between the ACES blobs and the newer transforms skip the OCES 
+      step.
+    - Doug: Display telemetry. New transforms go through that but don't break 
+      it out as external interface in CTL, for historical reasons.
+    - Mark: That will make it difficult to make generator which looks at CTL to 
+      generate. Not married to that plan but seems sensible for tracking 
+      upstream. Also happy to abandon it for config expressed in code if that's 
+      more useful.
+    - Doug: Was thinking could use generator to identify the ACES end and 
+      display code value end, but not necessarily use the OCES interface. For 
+      output transforms, some go direct from ACES and others through OCES, but 
+      we could handle them as if they went from ACES to device.
+    - Mark: At moment, generator doesn't parse CTL code, just looks at URN in 
+      header of files, so no way to track transforms or chains between functions.
+    - Doug: We know anything that expects OCES as input wants to include RRT.
+    - Mark: So use RRT as point?
+    - Doug: Part of problem is the RRT does not produce display referred 
+      imagery, which has caused confusion. Produces idealized output referred 
+      intermediate. Actual display referred colorimetry requires first part of 
+      ODT. RRT doesn't provide technical benefit. Think we can ignore RRT, and 
+      for transforms with OCES input, go from ACES to OCES.
+    - Mark: Not sure benefit to tracking CTL to get these relationships if we 
+      need to implement transforms. Maybe skip checking of CTL and 
+      programmatically build transforms.
+    - Michael: Could use generator for testing if not for config generation. To 
+      do image comparison etc.
```

Comment:
The intent of that generator is really to generate the Reference config as close as possible to the Book, i.e. CTL, we always talked about two configs, the Reference and the Studio one, the latter could build ontop of the former ideally.